rmgpy/rmg/modelTest.py

```diff
@@ -182,7 +190,64 @@ def testMakeNewReaction(self):
                     counter += len(rxnList)
 
         self.assertEquals(counter, 3)
+    
+    def testThermodynamicFiltering(self):
```

Comment:
I've split it into two unittests, one tests the new thermoFilterSpecies and one tests thermoFilterDown.